python/lsst/source/injection/inject_base.py

```diff
@@ -117,6 +117,16 @@ class BaseInjectConfig(PipelineTaskConfig, pipelineConnections=BaseInjectConnect
         doc="String to prefix to the entries in the *col_stamp* column, for example, a directory path.",
         default="",
     )
+    add_noise = Field[bool](
+        doc="Whether to randomly vary the injected flux in each pixel by an amount consistent with "
+        "the injected variance.",
+        default=True,
+    )
```

Comment:
We'd need to edit the docs on this ticket inside `doc/` to make clear this change of default behavior. I'd be happy to help on that front.

Reply:
Any help on that front would be appreciated.

Reply:
I'd recommend adding a new paragraph into https://github.com/lsst/source_injection/blob/main/doc/lsst.source.injection/reference/21_inject_synthetic_sources.rst.

This can mention not only the new variance injection capabilities, but also the noise injection too.